kytos/core/controller.py

```diff
@@ -606,7 +606,7 @@ async def event_handler(self, buffer_name: str):
         while True:
             try:
                 event = await event_buffer.aget()
-                self.notify_listeners(event)
+                await self.loop.create_task(self.notify_listeners(event))
```

Comment:
It could also be a `await asyncio.sleep(0)` instead of making `notify_listeners()` async.

Reply:
Using async now.